README.md

```diff
@@ -99,26 +99,53 @@ cargo near build
 
 Builds a NEAR smart contract along with its [ABI](https://github.com/near/abi) (while in the directory containing contract's Cargo.toml).
 
-By default, this runs a reproducible build in a [Docker](https://docs.docker.com/) container, which:
+Running the above command opens a menu with following variants: 
+
+### `non-reproducible-wasm`
+
+This is a regular build, which behaves much like and is a thin wrapper around a regular `cargo build --target wasm32-unknown-unknown --release`.
+
+Additional flags for build configuration can be looked up by
+
+```bash
+cargo near build non-reproducible-wasm --help
+```
+if needed.
+
+### `reproducible-wasm`
+
+This variant runs a reproducible build in a [Docker](https://docs.docker.com/) container, which:
```

Comment:
Let's explicitly recommend non-reproducible-wasm for local development and reproducible-wasm for production releases